docs/setup/tds-core-globals.js

```diff
@@ -12,10 +13,10 @@ import Strong from '@tds/core-strong'
 import Text from '@tds/core-text'
 import UnorderedList from '@tds/core-unordered-list'
 import Image from '@tds/core-image'
-import Button from '@tds/core-button'
 
 Object.assign(global, {
   Box,
+  Button,
```

Comment:
Please delete this to remove linting errors.

Reply:
> removed